src/npmPackage.js

```diff
@@ -161,6 +211,91 @@ export default function npmPackageFactory(rootDir = '', quiet = true) {
+            //      For example `@oat-sa-private/ui-core@5.19.0`.
+            //  2) `npx lerna version --no-git-tag-version --yes --conventional-commits --changelog-preset @oat-sa/conventional-changelog-tao --no-changelog`
+            //      It will calulate conventional-commit bump for each package,
+            //      and update all needed things in package.json/package-lock.json files.
+            //      `@oat-sa/conventional-changelog-tao` preset should be installed in the repo root, or otherwise can reference module from tao-extension-release:
+            //      `path.join(path.dirname(fileURLToPath(import.meta.url)),'../node_modules/@oat-sa/conventional-changelog-tao/index.js')`
+            //  3) delete local tags
+            for (const packageInfo of packagesInfo) {
+                // eslint-disable-next-line no-unused-vars
+                const { readme, _id, ...packageJson } = await readPkg({ cwd: packageInfo.packagePath });
+                packageJson.version = packageInfo.version;
+                for (const depPackageInfo of packagesInfo) {
+                    for (const depListKey of ['dependencies', 'devDependencies']) {
+                        if (packageJson[depListKey] && packageJson[depListKey][depPackageInfo.packageName]) {
+                            packageJson[depListKey][depPackageInfo.packageName] = `^${depPackageInfo.version}`;
+                        }
+                    }
+                }
+                await writePkg(packageInfo.packagePath, packageJson);
+
+                const packageLockJson = await this.readPackageLock(packageInfo.packagePath);
+                packageLockJson.version = packageInfo.version;
+                if ((packageLockJson.packages || [])['']?.name === packageInfo.packageName) {
+                    packageLockJson.packages[''].version = packageInfo.version;
+                }
+                await this.writePackageLock(packageInfo.packagePath, packageLockJson);
+            }
+
+            return this.npmCommand('i');
+        },
+
+        /**
+          * For monorepo,
+          * get information about its packages
+          * @return {Promise<Object[]} - `[{ packageName: string, packagePath: string, lastVersion: string, dependencies: string[] }]`
+          */
+        async lernaGetPackagesList() {
+            const packageListStr = await this.npxCommand('lerna list --json', { stdio: 'pipe' });
+
+            const packageListJson = JSON.parse(packageListStr);
+            const packagesInfo = packageListJson.map(packageInfo => ({
+                packagePath: path.relative(rootDir, packageInfo.location),
+                packageName: packageInfo.name,
+                lastVersion: packageInfo.version
+            }));
+
+            const packageGraphStr = await this.npxCommand('lerna list --graph', { stdio: 'pipe' });
```

Comment:
These commands seem to return only packages published to npm.
But we need all packages, because tao-deliver-fe has deliver-app and previewer-app.

See version bump https://github.com/oat-sa/tao-deliver-fe/pull/1074/commits/bf179f3b298daeb693859171d740cee303819b0d
From release done using this version from PR: https://github.com/oat-sa/tao-deliver-fe/pull/1074
Tool output was:
```
Reason: fixed bump to "minor"
npx lerna list --json
npx lerna list --graph
Monorepo package versions:
  Package: @oat-sa-private/tao-deliver-apps-common
    Last version found: 1.5.0
    Recommended version from commits: 1.6.0
    Reason: fixed bump to "minor"
  Package: @oat-sa-private/tao-item-runner-qtinui
    Last version found: 8.45.0
    Recommended version from commits: 8.46.0
    Reason: fixed bump to "minor"
  Package: @oat-sa-private/tao-test-runner-qtinui
    Last version found: 7.30.0
    Recommended version from commits: 7.31.0
    Reason: fixed bump to "minor"
```
Command was:
```
taoRelease npmReleaseMonorepo --base-branch release-2024-03 --release-branch main --release-tag 2024.03 --conventional-bump-type minor
```